### ISSUE TITLE
RadStation Engineering Needs Improvement BADLY

### ISSUE THREAD

**Issue by squirrelanna** [Simple-Station/Einstein-Engines#1126]
Overall, the station looks awesome and I want to enjoy it, but... The engineering department is a nightmare. Specifically, atmospherics and the supermatter chamber, the latter of which is rather glaring. I'll start there.

You cannot modify the inner chamber, the part that contains the extremely dangerous crystal, without almost entirely deconstructing its containment. The directional windows by the radiation collectors are aligned towards the inside of the chamber, which means:

1. You cannot change out the passive waste vents for scrubbers, because you cannot pry up the tiles without deconstructing the windows.

2. You cannot move the grounding rods, because if you unanchor them, you cannot anchor them again without deconstructing the windows.

3. You cannot add the tesla coils for additional power generation, because again, you cannot anchor them without deconstructing the windows.

This is on top of the flow of the pipes seemingly hellbent on getting pressure clogged without a LOT of finicking with volumetric flow, a problem that would be solved rather easily if certain modifications could be made without fundamentally deconstructing containment, like the aforementioned scrubbers in point number 1. 

Related to this, atmospherics could use some love... As it stands, it's a pipe-spaghetti hellscape that is daunting at best, obtuse at worst. The color coding does help some, but I pray for my IRL colorblind atmosians out there because I cannot fathom how it could be reasonably expected of them to decipher the logic without painstakingly following every pipe from start to finish and hoping you didn't get lost in the overlap along the way.

And then there's the TEG, the total opposite of the overdesigned distro/waste spaghetti just to its south. I'm no stranger to barebones round-start TEGs. But at least Saltern gives us something to build while still providing us with the BARE minimum in terms of mapping conventions. Namely, a burn chamber and a cold loop into space. Neither seem to be provided here. You have to build it all from scratch which wouldn't be so bad--it might even be really neat--if the supermatter chamber didn't ALSO need fundamental redesigning to make normally simple adjustments

Not to mention some baffling mapping choices made, like a connector port for a canister underneath an SMES near what looks to be a vestigial burn chamber entirely unrelated to the TEG. Almost certainly an honest mistake, but it belies an overall lack of care put into the design changes needed when porting this map from SS13, both to make it playable AND live up to the basic mapping standards the other maps have shown. Other examples include that SMES not even charging and the APC in the southwest corner of the supermatter chamber fully charged, yet connected to absolutely nothing.

Thanks for coming to my TedTalk

TL;DR: Please fix the facing of the directional windows in the supermatter chamber and give the whole department a second look because man... It is not well designed.

And here's a picture of a confused green robot girl trying to decipher what exactly the connector port is extracting from the SMES.
![image](https://github.com/user-attachments/assets/f2a74c50-99dc-4402-9975-9adcebfcaf23)

**Comment by Ichaie** [comment]
I will fix these things, but there is no guarantee that there will be any major changes to the atmos pipes, but I will try to make them a little less confusing.

**Comment by squirrelanna** [comment]
> I will fix these things, but there is no guarantee that there will be any major changes to the atmos pipes, but I will try to make them a little less confusing.

They by no means need to be perfect, for what it's worth! Just sometimes less is better, and in this case the more complex mixing might be best left to be designed by the atmos techs on staff, and with other frustrations fixed said techs should end up with more time to flex that creativity.
